metricbeat/module/ceph/osd_tree/_meta/fields.yml

```diff
@@ -0,0 +1,57 @@
+- name: osd_tree
+  type: group
+  description: >
+    ceph osd tree info
+  fields:
+    - name: id
+      type: long
+      description: >
+        osd or bucket node id
+    - name: name
+      type: string
```

Comment:
This can either be `keyword` or `text`. You want to use `keyword` on fields like this, with a limited number of possible values, and `text` for free form strings